aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -146,49 +156,111 @@ impl AggregatorChangeSet {
 mod test {
     use super::*;
     use aptos_aggregator::aggregator_extension::aggregator_id_for_test;
-    use claims::assert_matches;
-    use move_table_extension::TableHandle;
-
-    struct EmptyStorage;
-
-    impl TableResolver for EmptyStorage {
-        fn resolve_table_entry(
-            &self,
-            _handle: &TableHandle,
-            _key: &[u8],
-        ) -> Result<Option<Vec<u8>>, anyhow::Error> {
-            Ok(None)
-        }
+    use aptos_language_e2e_tests::data_store::FakeDataStore;
+    use aptos_types::state_store::state_key::StateKey;
+    use claims::{assert_matches, assert_ok};
+
+    pub fn serialize(value: &u128) -> Vec<u8> {
```

Comment:
There is `serialize` function  in `aggregator_extension.rs` which you can reuse :)

Reply:
Done.